sample-mail/src/main/java/com/hannesdorfmann/mosby3/sample/mail/MailApplication.java

```diff
@@ -18,6 +18,11 @@
 
   @Override public void onCreate() {
     super.onCreate();
+    if (LeakCanary.isInAnalyzerProcess(this)) {
```

Comment:
Yeah, my bad. This should be added to all examples that use LeakCanary. Could you please add that to the other examples too?